### PR TITLE
www: publication plot

### PR DESCRIPTION
**Pull request by tjhei**
This adds a javascript-generated bar plot on top of the publication list. Stolen from deal.II and adapted.

Preview:
image

### CODE REVIEW

**Comment by tjhei** [comment]
@bangerth if this looks good, we can put the same code inside deal.II and push the .html to the website (manually at first)

**Comment by tjhei** [comment]
@jppelteret FYI.

**Comment by jppelteret** [comment]
It looks great! :-) +1 for using this format on the deal.II website.

File: jabref-template/listrefs.end.layout [hunk 1]
```diff
@@ -13,4 +13,75 @@
...
+	else if (allrows[i].className.indexOf("bibtex")!=-1)
+	{
+	  // skip bibtex
+	}
+	else if (allrows[i].className.indexOf("abstract")!=-1)
+	{
+	  // skip abstract
+	}
+	else
+	    console.log("unknwon: " + allrows[i].className);
```
Comment (gassmoeller) [inline jabref-template/listrefs.end.layout]: `unknown`

File: jabref-template/listrefs.end.layout [hunk 2]
```diff
@@ -13,4 +13,75 @@
...
+    chart.draw(data, options);
+    }
+
+    var n_publications=0;
+    for (var i=first_year; i<=last_year; ++i)
+      n_publications += table[i-first_year+1][1];
+    document.getElementById("n_publications").innerHTML = "Total number of publications: <b>" + n_publications + "</b>.";
+  </script>
+
 </html>
```
Comment (bangerth) [inline jabref-template/listrefs.end.layout]: Missing endline (preexisting, but might as well fix it since you had that typo above...)

**Comment by tjhei** [comment]
> Do you want to test anything else, or is this essentially ready?

this is ready.

**Comment by gassmoeller** [comment]
Ok, lets give it a go.

**Pull request merged**